src/files/plugins/check_resources.py

```diff
@@ -308,6 +308,19 @@ def mechanism_warning_ids(connection, resource_type) -> Dict[str, str]:
         ]
         for ip in not_assigned_ips:
             warn_ids[ip] = "unassigned"
+
+    if resource_type == "port":
+        # Move DOWN ports from CRITICAL to warning if the instance was shutoff
+        all_ports = RESOURCES[resource_type](connection)
+        for port in all_ports:
+            try:
+                if port.status == "DOWN":
+                    server = connection.compute.get_server(port.device_id)
+                    if server.power_state == 4:  # 4 is SHUTOFF state
+                        warn_ids[port.id] = "SHUTOFF"
+            except openstack.exceptions.ResourceNotFound:
```

Comment:
**suggestion (non-blocking):** 

Can you provide some comment here why we ignore the error?

Reply:
updated with a comment now, thanks.